Composer/packages/client/src/pages/home/index.tsx

```diff
@@ -121,7 +121,7 @@ const Home: React.FC<RouteComponentProps> = () => {
         },
         onClick: () => {
           setCreationFlowStatus(CreationFlowStatus.SAVEAS);
-          navigate(`projects/${state.projectId}/save`);
+          navigate(`projects/${state.projectId}/${state.templateId}/save`);
```

Comment:
I wonder if `projects/${state.projectId}/save/${state.templateId}/` makes more meaning than `projects/${state.projectId}/${state.templateId}/save/`

Basically  `http://localhost:3000/projects/8012.804143209351/save/EchoBot/` instead of `http://localhost:3000/projects/8012.804143209351/EchoBot/save`

Since we are saving the project with a different template. But, I'm fine with the current url too if no one else has opinions